Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -80,6 +80,14 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
         .typingIndicatorView
         .init()
         .withoutAutoresizingMaskConstraints
+    
+    /// A button to scroll the collection view to the bottom.
+    ///
+    /// Visible when there is unread message and the collection view is not at the bottom already.
+    public private(set) lazy var scrollToLatestMessageButton: UIButton = components
```

Comment:
```suggestion
    open private(set) lazy var scrollToLatestMessageButton: UIButton = components
```

Reply:
@evsaev the `typingIndicatorView` above is also `public`, should I change it to `open` as well?

Reply:
ah yes, all of this should be `open`